lib/subscriptions_test_kit/suites/subscriptions_r5_backport_r4_server/subscription_rejection/reject_subscription_cross_version_extension_test.rb

```diff
@@ -0,0 +1,71 @@
+require_relative '../common/subscription_creation'
+
+module SubscriptionsTestKit
+  module SubscriptionsR5BackportR4Server
+    class RejectSubscriptionCrossVersionExtensionTest < Inferno::Test
+      include SubscriptionCreation
+
+      id :subscriptions_r4_server_reject_subscription_cross_version_extension
+      title 'Server Handles Unsupported Cross-Version Extensions'
+      description %(
+        When processing a request for a Subscription a server SHOULD verify that the Subscription is supported and does
+        not contain any information not implemented by the server. If the Subscription is no supported, the server
+        should reject the Subscription create request, or it should attempt to adjust the Subscription. Since the FHIR
+        R5 is currently under development, there are no guarantees these extensions will meet the requirements of
+        this guide. In order to promote widespread compatibility, cross version extensions SHOULD NOT be used
+        on R4 subscriptions to describe any elements.
+
+        The test will pass if the server either
+        1. rejects the Subscription by responding with a non-201 response, or
+        2. updates the Subscription resource to remove or replace the unsupported value.
+      )
+
+      verifies_requirements 'hl7.fhir.uv.subscriptions_1.1.0@12'
```

Comment:
I think we don't have a requirement for this one, so can clear the requirement out. Fine to still have the test, I think, but it won't be associated with a requirement.

Reply:
Removed the requirement but left the left in. See below on whether we'd like to keep the test!